src/apps/explorer/components/TransactionsTableWidget/index.tsx

```diff
@@ -38,17 +63,16 @@ export const TransactionsTableWidget: React.FC<Props> = ({ txHash }) => {
     return <Spinner spin size="3x" />
   }
 
-  // TODO replace with a real table component
   return (
-    <>
-      <h1>
-        Transaction details
-        <TitleAddress
-          textToCopy={txHash}
-          contentsToDisplay={<BlockExplorerLink type="tx" networkId={networkId} identifier={txHash} />}
-        />
-      </h1>
```

Comment:
@maria-vslvn  It would be better to leave this inside the `TransactionTableWidget` because as @elena-zh  mentioned if I search for a transaction with the [correct format but it doesn't exist](https://pr882--gpui.review.gnosisdev.com/gc/search/0x3f1aac7fb6eef92e253defb065e39873cc494ef5f4ccfb2644792c7aba937856/), we would be showing this header while searching for the order (even though it does

Reply:
yes, sure